webpack/modules/aws-cdk-lib/core/lib/asset-staging.js

```diff
@@ -0,0 +1,41 @@
+const code = require("../../../../../node_modules/aws-cdk-lib/core/lib/asset-staging.js");
+const fs = require("fs");
+
+class WebAssetStaging extends code.AssetStaging {
+  bundle(options, bundleDir) {
+    if (fs.existsSync(bundleDir)) {
+      return;
+    }
+    fs.mkdirSync(bundleDir, { recursive: true });
+    let localBundling;
+    try {
+      console.log(`Bundling asset ${this.node.path}... in ${bundleDir}\n`);
```

Comment:
avoid using `console.*` please. `debug` should be used instead. we don't want logs in browser by default: https://github.com/3p3r/cdk-web/blob/main/docs/logs.md

Reply:
Like another tool we may or may not have made I think we need a logger . This log is part of cdk core. I will change to debug though. https://github.com/aws/aws-cdk/blob/v1.152.0/packages/@aws-cdk/core/lib/asset-staging.ts#L444 it is written to stderr.

Reply:
wowow I can't believe they're directly writing to stderr. cdk has a logger that I patched. might have been in the cli repo. anyhoo, calls to console.log and process.std** streams are all nuked by webpack FYI.